benchpress/benchpress/lib/job.py

```diff
@@ -103,6 +103,17 @@ def run(self):
                 output = 'stdout:\n{}\nstderr:\n{}'.format(stdout, stderr)
                 cmd = ' '.join(cmd)
                 raise CalledProcessError(process.returncode, cmd, output)
+
```

Comment:
Would be nice if this would be in a function, but this is still good.

Reply:
Noted, if this gets any messier next time I'll refactor.